### PR TITLE
✨ Add helper function

### PR DESCRIPTION
**Pull request by beranm14**
Related: #64945

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -31,3 +33,8 @@ RUN wget -q https://releases.hashicorp.com/vault/1.8.4/vault_1.8.4_linux_amd64.z
     rm vault.zip
 
 COPY --from=docker/compose:alpine-1.27.4 /usr/local/bin/docker-compose /usr/local/bin/
+
+# There is no globing in URL for ADD command
```
Comment (beranm14) [inline Dockerfile:37]: not needed comment tho

Comment (tomashejatko) [reply Dockerfile:37]: +1

File: Dockerfile [hunk 2]
```diff
@@ -31,3 +33,8 @@ RUN wget -q https://releases.hashicorp.com/vault/1.8.4/vault_1.8.4_linux_amd64.z
...
+ADD https://raw.githubusercontent.com/AckeeDevOps/gitlab-ci-utils/$GITLAB_CI_UTILS_VERSION/scripts/helper_functions.sh /usr/local/bin/helper_functions.sh
+
+RUN chmod +x /usr/local/bin/*
```
Comment (tomashejatko) [inline Dockerfile:40]: Can now be changed only to `/usr/local/bin/helper_functions.sh`

**Comment by tomashejatko** [comment]
LGTM

**Review by tomashejatko — APPROVED** [review]

**Pull request merged**